<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestOwen</class>
 <widget class="QWidget" name="TestOwen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestOwen</string>
  </property>
  <widget class="QGroupBox" name="grpVid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rdbSerial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbUSB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>USB</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spbNum">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/dev/ttyS0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOpen">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>112</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть порт</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatNam">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Состояние</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStat">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>неопределен</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>112</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть порт</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxSpeed">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>140</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>13</number>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>134</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spdIdx">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>220</y>
     <width>58</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbAddr">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>180</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1023</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeed">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>146</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Скорость</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAddr">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>187</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIdx">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>225</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Индекс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdConnect">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>140</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
